### PR TITLE
ShiftEntry

### PR DESCRIPTION
**Pull request by MyIgel**
This PR adds the `ShiftEntry` model which stores its data in the `shift_entries` table.

:exclamation: The two commits must be squashed before merging :exclamation:

### CODE REVIEW

**Comment by MyIgel** [comment]
:wave: @weeman1337

File: src/Models/Shifts/ShiftEntry.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    protected $fillable = [ // phpcs:ignore
+        'shift_id',
+        'angel_type_id',
+        'user_id',
+        'user_comment',
+        'freeloaded',
+        'freeloaded_comment',
+    ];
+
+    /** @var array<string, string> default attributes */
```
Comment (weeman1337) [inline src/Models/Shifts/ShiftEntry.php:47]:
```suggestion
    /** @var array<string, string | bool> default attributes */
```

File: tests/Unit/Models/AngelTypeTest.php [hunk 2]
```diff
@@ -61,6 +62,19 @@ public function testUserAngelTypes(): void
         $this->assertCount(2, $angeltypes);
     }
 
+    /**
+     * @covers \Engelsystem\Models\AngelType::shiftEntries
+     */
+    public function testShiftEntries(): void
+    {
+        $angelType = new AngelType(['name' => 'test type']);
```
Comment (weeman1337) [inline tests/Unit/Models/AngelTypeTest.php:70]:
```suggestion
        $angelType = AngelType::create(['name' => 'test type']);
```

ohne `save()` dann

**Review by weeman1337 — APPROVED** [review]
:+1:

**Review by weeman1337 — APPROVED** [review]
:+1:

**Pull request merged**